src/CSnakes.Runtime/CPython/Long.cs

```diff
@@ -16,6 +16,9 @@ internal unsafe partial class CPythonAPI
     [LibraryImport(PythonLibraryName)]
     internal static partial long PyLong_AsLongLong(PyObject p);
 
+    [LibraryImport(PythonLibraryName)]
+    internal static partial int PyLong_AsLong(PyObject p);

```

Comment:
Ah ha! This is part of a previous issue I had. This signature is only correct on Windows. Reading the [API](https://docs.python.org/3/c-api/long.html#c.PyLong_AsLong), this returns a C `long`. The details are sad and I'll just point you at the [documentation](https://learn.microsoft.com/dotnet/standard/native-interop/best-practices#cc-long).

Reply:
I'll handle that in another issue I think. I'd thought it was optimistic to use `long`